pkg/kubectl/pdb_test.go

```diff
@@ -0,0 +1,338 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"reflect"
+	"testing"
+
+	policy "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func TestPodDisruptionBudgetV1Generate(t *testing.T) {
+	name := "foo"
```

Comment:
I'm a little bit pedantic about that (and maybe it's just me, so feel free to dismiss), but I really like hard-coded values in tests (as opposed to variables). I think it makes it much clearer what's expected and what's going on. I don't have the overhead in my head of remembering the values of each of these variables.

Reply:
I don't have a strong opinion here, it's just that I'm used to vars :) Is there any k8s best practices around this? let me know if you want me to change it to hard-coded values, please.

Reply:
No, I don't think we have rules nor that it does matter that much. Don't worry about that.